test/module/irohad/validation/chain_validation_test.cpp

```diff
@@ -15,105 +15,161 @@
  * limitations under the License.
  */
 
+#include "builders/protobuf/block.hpp"
 #include "module/irohad/ametsuchi/ametsuchi_mocks.hpp"
 #include "module/irohad/model/model_mocks.hpp"
 #include "validation/impl/chain_validator_impl.hpp"
 
+// TODO: 14-02-2018 Alexey Chernyshov remove after relocation to shared_model
+#include "backend/protobuf/from_old_model.hpp"
+
 using namespace iroha;
 using namespace iroha::model;
 using namespace iroha::validation;
 using namespace iroha::ametsuchi;
 
-using ::testing::_;
 using ::testing::A;
 using ::testing::ByRef;
 using ::testing::InvokeArgument;
 using ::testing::Return;
+using ::testing::_;
 
 class ChainValidationTest : public ::testing::Test {
  public:
   void SetUp() override {
     validator = std::make_shared<ChainValidatorImpl>();
     storage = std::make_shared<MockMutableStorage>();
     query = std::make_shared<MockWsvQuery>();
-
-    peer.pubkey.fill(2);
     peers = std::vector<Peer>{peer};
+  }
 
-    block.sigs.emplace_back();
-    block.sigs.back().pubkey = peer.pubkey;
-    block.prev_hash.fill(0);
-    hash = block.prev_hash;
+  /**
+   * Get block builder to build blocks for tests
+   * @return block builder
+   */
+  auto getBlockBuilder() const {
+    return shared_model::proto::BlockBuilder()
+        .transactions(std::vector<shared_model::proto::Transaction>{})
+        .txNumber(0)
+        .height(1)
+        .prevHash(hash)
+        .createdTime(iroha::time::now());
```

Comment:
What is the purpose to use system time for the test? Maybe set static value?

Reply:
what value? All tests use `iroha::time::now()`.

Reply:
I think here we can put any value. `iroha::time::now()` is fine

Reply:
Well, in this case, better to write the static value. But, if the value is not used what is the reason to set it explicitly?

Reply:
I think it better just use `BlockBuilder` instead the test one